docs/dr-health.md

```diff
@@ -0,0 +1,42 @@
+# Disaster Recovery
+
+DR health command is used to get connection status of one cluster from another cluster in mirroring enabled clusters. We fetch cephblockpool with mirrroing enabled if not found it will exit with relevant logs. Optionally, it accept all the args that are supported with `ceph status` command like `--debug-ms 1`.
+
+Example: `kubectl rook-ceph dr health [ceph status args]`
+
+## Command
```

Comment:
In the future there may be multiple DR commands, so we can have one section for each. The above paragraph and example would move inside this header.
```suggestion
## Health
```

Reply:
Agree!